pkg/scalers/azure_eventhub_scaler.go

```diff
@@ -175,12 +164,6 @@ func parseCommonAzureEventHubMetadata(config *scalersconfig.ScalerConfig, meta *
 	}
 	meta.eventHubInfo.ServiceBusEndpointSuffix = serviceBusEndpointSuffix
 
-	activeDirectoryEndpoint, err := azure.ParseActiveDirectoryEndpoint(config.TriggerMetadata)
-	if err != nil {
-		return err
-	}
-	meta.eventHubInfo.ActiveDirectoryEndpoint = activeDirectoryEndpoint
```

Comment:
Same as above, I think that this is now managed by the SDK. When we used rest client we needed to provide more info